vs-csproj/NugetLightProvider.csproj

```diff
@@ -63,47 +63,104 @@
+	<Compile Include="..\Package\PackageItem.cs" />
+	<Compile Include="..\Package\PackageName.cs">
+      <SubType>Code</SubType>
+    </Compile>
+	
+	<Compile Include="..\Repository\IPackageRepository.cs" />
     <Compile Include="..\Repository\IPackageRepositoryFactory.cs" />
     <Compile Include="..\Repository\LocalPackageRepository.cs" />
-    <Compile Include="..\Package\PackageBase.cs" />
-    <Compile Include="..\NugetLightClient.cs" />
-    <Compile Include="..\Package\PackageDependency.cs" />
-    <Compile Include="..\Utility\NuGetPathUtility.cs" />
-    <Compile Include="..\Utility\PackageUtility.cs" />
-    <Compile Include="..\Package\PackageItem.cs" />
+	<Compile Include="..\Repository\NuGetPackageRepository.cs" />
+	<Compile Include="..\Repository\NuGetSearchContext.cs" />
+	<Compile Include="..\Repository\NuGetSearchTerm.cs" />
+	<Compile Include="..\Repository\PackageEntryInfo.cs" />
+	<Compile Include="..\Repository\PackageRepositoryCreateParameters.cs" />
     <Compile Include="..\Repository\PackageRepositoryFactory.cs" />
-    <Compile Include="..\PackageSource.cs" />
-    <Compile Include="..\NugetLightRequest.cs">
-      <SubType>Code</SubType>
+    
+    <Compile Include="..\Properties\AssemblyInfo.cs" />
+    
+    <Compile Include="..\gen\Microsoft.PackageManagement.NuGetProvider.Resources.Messages.cs">
+      <AutoGen>True</AutoGen>
+      <DesignTime>True</DesignTime>
+      <DependentUpon>Messages.resx</DependentUpon>
     </Compile>
-    <Compile Include="..\..\oneget\src\Microsoft.PackageManagement\Providers\Inbox\Common\Utility\PathUtility.cs" />
+    
+    <Compile Include="..\Sdk\Constants.cs" />
+    <Compile Include="..\Sdk\ErrorCategory.cs" />
+	<Compile Include="..\Sdk\IRequestOutput.cs" />
+	<Compile Include="..\Sdk\NuGetRequestOutput.cs" />
+	<Compile Include="..\Sdk\NullRequestOutput.cs" />
+    <Compile Include="..\Sdk\Request.cs" />
+    <Compile Include="..\Sdk\RequestWrapper.cs" />
+	
+	<Compile Include="..\Utility\CatalogUrlConverter.cs" />
+	<Compile Include="..\Utility\ConcurrentInMemoryCache.cs" />
+	<Compile Include="..\Utility\DependencyGroupConverter.cs" />
+	<Compile Include="..\Utility\DynamicJsonParser.cs" />
+	<Compile Include="..\Utility\HttpQueryBuilder.cs" />
+	<Compile Include="..\Utility\IDynamicJsonObjectConverter.cs" />
+    <Compile Include="..\Utility\NuGetPathUtility.cs" />
+	<Compile Include="..\Utility\NuGetWebUtility.cs" />
+	<Compile Include="..\Utility\PackageBaseConverter.cs" />
+	<Compile Include="..\Utility\PackageDependencyConverter.cs" />
+	<Compile Include="..\Utility\PackageFilterUtility.cs" />
+    <Compile Include="..\Utility\PackageUtility.cs" />
+    <Compile Include="..\Utility\ProgressTracker.cs" />
+	<Compile Include="..\Utility\PublicObjectView.cs" />
+    <Compile Include="..\Utility\TaskGroup.cs" />
+    
+    
+    <!--<Compile Include="..\..\oneget\src\Microsoft.PackageManagement\Providers\Inbox\Common\Utility\PathUtility.cs" />
```

Comment:
Should we just remove them instead of commenting them out?

Reply:
b461de0